panda/src/navigation/navMeshNode.h

```diff
@@ -0,0 +1,55 @@
+/**
+ * PANDA 3D SOFTWARE
+ * Copyright (c) Carnegie Mellon University.  All rights reserved.
+ *
+ * All use of this software is subject to the terms of the revised BSD
+ * license.  You should have received a copy of this license along
+ * with this source code in a file named "LICENSE."
+ *
+ * @file navMeshNode.h
+ * @author ashwini
+ * @date 2020-060-21
+ */
+
+#ifndef NAVMESHNODE_H
+#define NAVMESHNODE_H
+
+#include <string>
+#include "geom.h"
+#include "pandaFramework.h"
+#include "pandaSystem.h"
+#include "navMesh.h"
+#include <string>
+
+class EXPCL_NAVIGATION NavMeshNode: public PandaNode
```

Comment:
Missing docstring

Reply:
Should docstrings be added in header files as well? I have added them in cxx files only for all the functions